saleor/site/utils.py

```diff
@@ -0,0 +1,19 @@
+from django.conf import settings
+
+from .models import SiteSetting
+
+
+def get_site_settings(request):
+    if not hasattr(request, 'site_settings'):
+        site_settings_id = getattr(settings, 'SITE_SETTINGS_ID', None)
+        request.site_settings = get_site_settings_uncached(site_settings_id)
+    return request.site_settings
+
+
+def get_site_settings_uncached(site_id=None):
+    return SiteSetting.objects.get(pk=site_id)
+
+
+def get_setting_value(request, key):
```

Comment:
We can serialize settings object to a dict. It will be helpful in caching. Also we will be able to do direct dictionary key lookups. Just like @mociepka shared in his snippet, we can use `model_to_dict` to serialize an object.